compilers/concrete-compiler/compiler/tests/python/test_compilation.py

```diff
@@ -298,13 +298,6 @@ def test_lib_compile_and_run_global_p_error(keyset_cache):
     _test_lib_compile_and_run_with_options(keyset_cache, options)
 
 
-def test_lib_compile_and_run_security_level(keyset_cache):
```

Comment:
expected change?

Reply:
yes, we kept only 128 bits at least for now

Reply:
ok thx